ui/js/component/walletSend/view.jsx

```diff
@@ -77,7 +81,7 @@ const WalletSend = props => {
           contentLabel={__("Transaction failed")}
           onConfirmed={closeModal}
         >
-          {__("Something went wrong")}:
+          {__(error)}
```

Comment:
Two questions here:
  1) Are we OK with displaying the error message from the daemon directly? Without an error code, my only other option would be to check the text string of the error message and display our own message based on that. Right now this is the message it's passing through from the daemon:
<img width="402" alt="screen shot 2017-09-04 at 6 44 16 pm" src="https://user-images.githubusercontent.com/7283989/30039985-90e15b2a-91a6-11e7-903c-69334cbb9847.png">

  2) In the meanwhile, should this be wrapped in an `__` function or not? Feels brittle since the message on the daemon side might change, OTOH we don't want to just always have it be in English.

Reply:
1) Yes, it is fine to display daemon error messages.
2) Do not i18n daemon errors. This is an unsolved problem.
3) This doesn't need to block PR (it already worked this way), but form errors should be inline when possible.